### DISCUSSION TITLE
Pipeline sync failure for Zendesk Chat source with S3 parquet destination

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34089] [General]
I have created a connection using the following:
Source: Zendesk Chat
Destination: S3 parquet 

Sync failure:

![image](https://github.com/airbytehq/airbyte/assets/106967803/d5681e58-8018-4fe9-8291-78f9c31b69e8)
![image](https://github.com/airbytehq/airbyte/assets/106967803/04dd192d-0111-4423-a660-f41bddab9fb2)

Logs are attached below:
[zendeskchat_parq_logs.txt](https://github.com/airbytehq/airbyte/files/13887051/zendeskchat_parq_logs.txt)

**Comment by rwask** [comment]
Hi @WisamulHaque 

The error suggests that there is an issue with converting JSON to Avro format, specifically with a field named 'session' as well as other fields listed in the logs. The Avro schema for this field expects it to be one of these types: NULL or STRING, forming a union. However, the actual data seems to violate this expectation.

Can you give us some more information on these fields are there any special characters that may be causing the problem going to S3 parquet destination?

**Comment by rwask** [comment]
@WisamulHaque Let us know if you are still running into any issues with this Sync. I will close this issue for now but feel free to reference this if you are still having issues after taking a look at my previous reply.
